### DISCUSSION TITLE
23.10.26 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#45] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 은서님, 우재님 코드 리뷰
    - 면접
- 내일은 어떤 일을 하나요?
    - 발표 준비 (다음 주에 집중해서 진도 나가겠습니다..🥺)

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 흩어진 Domain 조립
* 내일은 어떤 일을 하나요?
  * 발표 준비

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 코드 리뷰
  - 비동기 적용 테스트 코드 작성
- 내일은 어떤 일을 하나요?
  - 발표 준비
